src/hotspot/share/services/memTracker.hpp

```diff
@@ -125,7 +125,7 @@ class MemTracker : AllStatic {
   }
 
   // Record arena size change. Arena size is the size of all arena
-  // chuncks that backing up the arena.
+  // chunks that backing up the arena.
```

Comment:
This doesn't read correctly (not due to your change). Perhaps "that back up" or "that are backing up" is what was intended.